src/plugins/data/server/search/strategies/sql_search/request_utils.test.ts

```diff
@@ -8,30 +8,26 @@
 
 import { getDefaultAsyncSubmitParams, getDefaultAsyncGetParams } from './request_utils';
 import moment from 'moment';
-import { SearchSessionsConfigSchema } from '../../../../config';
-
-const getMockSearchSessionsConfig = ({
-  enabled = true,
-  defaultExpiration = moment.duration(7, 'd'),
-} = {}) =>
-  ({
-    enabled,
-    defaultExpiration,
-  } as SearchSessionsConfigSchema);
+import { getMockSearchConfig } from '../../../../config.mock';
 
 describe('request utils', () => {
   describe('getDefaultAsyncSubmitParams', () => {
     test('Uses `keep_alive` from default params if no `sessionId` is provided', async () => {
-      const mockConfig = getMockSearchSessionsConfig({
-        defaultExpiration: moment.duration(3, 'd'),
+      const mockConfig = getMockSearchConfig({
+        sessions: {
+          defaultExpiration: moment.duration(3, 'd'),
+        },
       });
       const params = getDefaultAsyncSubmitParams(mockConfig, {});
-      expect(params).toHaveProperty('keep_alive', '1m');
+      expect(params).toHaveProperty('keep_alive', '60000ms');
     });
 
-    test('Uses `keep_alive` from config if enabled and session is stored', async () => {
-      const mockConfig = getMockSearchSessionsConfig({
-        defaultExpiration: moment.duration(3, 'd'),
+    // unskip when SQL has full session support https://github.com/elastic/kibana/issues/127880
+    test.skip('Uses `keep_alive` from config if enabled and session is stored', async () => {
```

Comment:
Previously the "force disabling of search sessions" for SQL happened on a level higher - in the sql_search_strategy. But in the current setup I think it is clearer to contain inside request_utils - hence adjusting tests